### DISCUSSION TITLE
Action To Cut Record to Another Resource

### DISCUSSION THREAD

**Discussion by ridone713** [filamentphp/filament#12000] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

![Screenshot (572)](https://github.com/filamentphp/filament/assets/164133045/33c74cb5-8985-4e74-a235-728fa62de655)
![Screenshot (571)](https://github.com/filamentphp/filament/assets/164133045/3ac1085c-b612-44ca-b4d1-993667397c95)
How do I cut records in the Tagihan table automatically to the Pembayaran menu after I press the 'Lunas' action button, and when I switch to the Pembayaran menu, the status changes automatically from 'Belum Lunas' to 'Lunas' in the Pembayran menu.

**Comment by leandrocfe** [comment]
https://github.com/filamentphp/filament/discussions/11952#discussioncomment-8849735
You can delete the record after replicating it

**Reply by ridone713** [reply]
it doesn't work sir

**Reply by ridone713** [reply]
Pembayaran resource doesn't have a table, I mean: Pembayaran resource and Tagihan resource has 1 table, I just want to move selected record from Tagihan resource to Pembayaran resource by click an action. after moving the record, enum value in the status column automatically change into another enum value at the Pembayaran resource. pleae sir help me, i've been stuck since 4 days ago

**Reply by leandrocfe** [reply]
Are you using one DB table and two Resources? Are you able to share this project on Github?
